ZenPacks/zenoss/LinuxMonitor/tests/plugindata/linux/build.zenoss.loc/ifconfig

```diff
@@ -1,4 +1,4 @@
-/sbin/ifconfig -a && echo __COMMAND__ && /bin/dmesg
+export PATH=$PATH:/sbin:/usr/sbin; if which ifconfig >/dev/null 2>&1; then ifconfig -a; elif which ip >/dev/null 2>&1; then echo "### ip addr output"; ip addr; else echo "No ifconfig or ip utilities were found."; exit 127; fi && echo __COMMAND__ && /bin/dmesg
```

Comment:
Nevermind my previous comment. I didn't realize this was a unit test and which is already being used in the ifconfig modeler in platform.